### PR TITLE
G7edits 070815

### PR DESCRIPTION
**Pull request by superroz**
Edits to reflect changes to the G7 application process and edits from CCS.

### CODE REVIEW

**Comment by quis** [comment]
The "published at" date needs updating.

File: app/templates/content/suppliers-guide-g-cloud.html [hunk 1]
```diff
@@ -56,69 +56,62 @@ <h2 id="about-g-cloud">
...
-  <div class="image-embedded"><img alt="" src="/static/images/supplying-process.png"></div>
-
   <h3 id="register">
-  <span class="number">2.1 </span>Sign up to the Crown Commercial Service (CCS) eSourcing Portal
+  <span class="number">2.1 </span>Create, or log into, a supplier account on the Digital Marketplace
   </h3>
 
   <p>
-  Start your application by signing up to the <a rel="external" href="https://crowncommercialservice.bravosolution.co.uk/web/login.shtml">CCS eSourcing Portal</a>.
+  Start your application by creating, or logging into <a rel="external" href="https://www.digitalmarketplace.service.gov.uk/suppliers/login">your supplier account</a>.
```
Comment (quis) [inline app/templates/content/suppliers-guide-g-cloud.html]: This isn't an external link any more.

**Comment by tombye** [comment]
Note: the publish date is set to tomorrow because that's when it will be deployed.

**Comment by tombye** [comment]
@quis that should cover all your points, can you have another look?

**Comment by quis** [comment]
No 5:30pm deploys? :crying_cat_face: 

:+1:

**Pull request merged**